src/display/displayHead.js

```diff
@@ -17,32 +16,27 @@ import font2 from "./font2.js";
 import meta from "./headMetaTags";
 import ifStyle from "./ifStyle.js";
 
-
 let addon1 = {
-
- blobHead, 
- bodyStyle,
- mediaStyle,
- style,
- style2,
- font1 ,
- font2 ,
- meta ,
- ifStyle,
-
-    // variable:value, 
-
-}
+  blobHead,
+  bodyStyle,
+  mediaStyle,
+  style,
+  style2,
+  font1,
+  font2,
+  meta,
+  ifStyle,
+
+  // variable:value,
+};
 
 //variant one
 const settings = {
-    component: headComponent,
-    params: addon1,
-
-}
+  component: headComponent,
+  params: addon1,
+};
 
 const Factory = new displayFactoryTwo();
 
-console.log(Factory.create(settings))
+console.log(Factory.create(settings));
```

Comment:
does it work as it supposes to?

Reply:
I mean is it returning an HTML with everything inside of it? because if it's working without errors - it's awesome!